doc/nrf/ug_thread_certification.rst

```diff
@@ -66,10 +66,15 @@ Complete the following steps to prepare for the certification tests:
 
    The :ref:`ot_cli_sample` sample is used as a base, modified with the :file:`harness/overlay-cert.conf` overlay file.
 
-   .. code-block::
+   * If building on the command line, use the following command:
 
-         cd ncs/nrf/samples/openthread/cli/
-         west build -b nrf52840dk_nrf52840 -- -DOVERLAY_CONFIG=harness/overlay-cert.conf -DCONFIG_OPENTHREAD_LIBRARY_1_1=y
+     .. code-block::
+
+        cd ncs/nrf/samples/openthread/cli/
+        west build -b nrf52840dk_nrf52840 -- -DOVERLAY_CONFIG=harness/overlay-cert.conf -DCONFIG_OPENTHREAD_LIBRARY_1_1=y
+
+   * If building using Visual Studio Code, you must first `create the application<Creating an application>`_ using the CLI sample, and then `build the application<Building an application>`_ and select the :file:`harness/overlay-cert.conf` overlay file in the :guilabel:`Kconfig fragment` drop-down menu.
```

Comment:
The part about using `CONFIG_OPENTHREAD_LIBRARY_1_1=y` is missing for the VSCode option.

Reply:
Ah, good catch. I think that would be added to the **Extra CMake arguments** field.